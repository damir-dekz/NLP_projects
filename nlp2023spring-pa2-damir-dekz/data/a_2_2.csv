Sport,Politics,Culture
200,20,30
10,150,15
60,90,10
