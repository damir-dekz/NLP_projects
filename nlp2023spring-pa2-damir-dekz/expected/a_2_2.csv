Category,Precision,Recall,F1-score
Sport,0.7407407407407407,0.8,0.7692307692307692
Politics,0.5769230769230769,0.8571428571428571,0.689655172413793
Culture,0.18181818181818182,0.0625,0.09302325581395349
